incubator/hackmd/templates/deployment.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "hackmd.fullname" . }}
+  labels:
+    app: {{ template "hackmd.name" . }}
+    chart: {{ template "hackmd.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "hackmd.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "hackmd.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /status
+              port: 3000
+            initialDelaySeconds: 120
+          readinessProbe:
+            httpGet:
+              path: /status
+              port: 3000
+            initialDelaySeconds: 30
+          env:
+            - name: HMD_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "hackmd.postgresql.fullname" . }}
+                  key: postgres-password
+            - name: HMD_DB_URL
+              value: postgres://{{ .Values.postgresql.postgresUser }}:$(HMD_DB_PASSWORD)@{{ template "hackmd.postgresql.fullname" . }}:5432/{{ .Values.postgresql.postgresDatabase }}
```

Comment:
At previous post, I mistook the DB_URL (I was using {{ template "hacked.fullname" . }} as DB host). Maybe it caused this problem?